vignettes/VectorControl_IRS.Rmd

```diff
@@ -0,0 +1,158 @@
+  legend('bottomleft', box.lty = 0, legend = c('Spraying', 'Prevalence for IRS scenario','Prevalence for control scenario'),
+         col = c('black', cols[3], cols[5]), lty = c(2,1,1), lwd = 2.5)
+}
+```
+
+### Parameterisation 
+
+Use the `get_parameters()` function to generate the list of parameters for a seasonal profile, accepting the default values to run the simulation from an equilibrium starting point. The seasonality will drive mosquito dynamics in the absence of interventions. 
+
+```{r}
+year <- 365
+month <- 30
+sim_length <- 3 * year
+human_population <- 1000
+starting_EIR <- 50
+
+simparams <- get_parameters(
+  list(
+    human_population = human_population,
+    # seasonality parameters
+    model_seasonality = TRUE, # Let's try a bi-modal model
+    g0 = 0.28605,
+    g = c(0.20636, -0.0740318, -0.0009293),
+    h = c(0.173743, -0.0730962, -0.116019),
+    # Here, we are asking for the output to contain information on severe incidence in children 2-10 years old.
+    clinical_incidence_rendering_min_ages = 2*year,
+    clinical_incidence_rendering_max_ages = 10*year
+  )
+)
+
+simparams <- set_equilibrium(simparams, starting_EIR)
+
+# Running simulation with no IRS
+output_control <- run_simulation(sim_length, simparams)
+```
+  
+## Simulation 
+
+Then we can run the simulation for a variety of IRS strategies. In the example below, there are two rounds of IRS at 80% coverage, each 3 months prior to peak rainfall for that year. The function `peak_season_offset()` outputs a timestep when peak rainfall will be observed. Notice that the matrices specified for the parameters for `ls_theta`, `ls_gamma`, `ks_theta`, `ks_gamma`, `ms_theta`, and `ms_gamma` have two rows, one for each timestep where IRS occurs, and 1 column for all mosquitoes, not divided by species.
+
+```{r, fig.align='center', fig.height=4, fig.width=6}
+peak <- peak_season_offset(simparams)
+
+sprayingtimesteps <- c(1, 2) * year + peak - 3 * month # There is a round of IRS in the 1st and second year 3 months prior to peak transmission.
+
+sprayingparams <- set_spraying(
+  simparams,
+  timesteps = sprayingtimesteps,
+  coverages = rep(.8, 2), # # Each round covers 80% of the population 
+  ls_theta = matrix(c(2.025, 2.025), nrow=2, ncol=1), # Matrix of mortality parameters
+  ls_gamma = matrix(c(-0.009, -0.009), nrow=2, ncol=1), # Matrix of mortality parameters per timestep
+  ks_theta = matrix(c(-2.222, -2.222), nrow=2, ncol=1), # Matrix of feeding success parameters
+  ks_gamma = matrix(c(0.008, 0.008), nrow=2, ncol=1), # Matrix of feeding success parameters per timestep
+  ms_theta = matrix(c(-1.232, -1.232), nrow=2, ncol=1), # Matrix of deterrence parameters
+  ms_gamma = matrix(c(-0.009, -0.009), nrow=2, ncol=1) # Matrix of deterrence parameters per timestep
+)
+
+output <- run_simulation(sim_length, sprayingparams)
+```
+
+
+### Plot prevalence
+```{r,fig.align='center', fig.height=5, fig.width=7}
+plot_prev()
+```
+
+## Additional species 
```

Comment:
This next section is really nice - but I think should be moved to a seperate vignette. It could focus on how we specify differnt species and binomics and then include the example of how this can alter intervention impact.

What do you think @htopazian ?

Reply:
Ok, so what I've decided is to use only the default species = All for the bed net and IRS vignettes. I added a small section about the possibility of modelling multiple species and referenced the new set species vignette (need to add link later), and have created this new vignette for set_species().